lib/std/Build/Step/Run.zig

```diff
@@ -963,8 +971,8 @@ fn spawnChildAndCollect(
     const arena = b.allocator;
 
     var child = std.process.Child.init(argv, arena);
-    if (self.cwd) |cwd| {
-        child.cwd = b.pathFromRoot(cwd);
+    if (self.cwd) |lazy_cwd| {
+        child.cwd = b.pathFromRoot(lazy_cwd.getPath(b));
```

Comment:
LazyPath handles relativeness to the build root:

```suggestion
        child.cwd = lazy_cwd.getPath(b);
```